advanced/am-pattern-2/templates/am/gateway/wso2am-pattern-2-gateway-ingress.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Chathuru can you please update the license header year in the newly added files? Please check the other files as well.

Reply:
Fix the issue in https://github.com/wso2/kubernetes-apim/pull/344/commits/c1d371588b9d9e50fa267a4bbf5a74bc7b50517b